adaptive_thresh_constant: 5.0
adaptive_thresh_win_size_max: 75
adaptive_thresh_win_size_min: 7
adaptive_thresh_win_size_step: 8
blur_strength: 3
clahe_clip_limit: 2.0
clahe_tile_size: 8
corner_refinement_max_iterations: 30
corner_refinement_min_accuracy: 0.05
corner_refinement_win_size: 7
error_correction_rate: 0.6
max_marker_perimeter_rate: 4.0
min_marker_perimeter_rate: 0.02
polygonal_approx_accuracy_rate: 0.05
reproj_rms_threshold: 3.0
